### DISCUSSION TITLE
Release 9.0.1-beta.1 is out on Nuget!

### DISCUSSION THREAD

**Discussion by holytshirt** [BrighterCommand/Brighter#1492] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/BrighterCommand/Brighter/releases/tag/9.0.1-beta.1'>9.0.1-beta.1</a>.</em>

**Comment by holytshirt** [comment]
The Version 9.0.1-beta.1 packages are on Nuget!
https://www.nuget.org/profiles/BrighterCommand

Release notes and Upgrade notes will be updated in the forthcoming days. 
This is a huge release with breaking changes, please test and feedback any issues.

**Comment by preardon** [comment]
@holytshirt Can we please get the Next Beta out this weekend After Ian has Merged his Fix Red Tests PR?
